Monika After Story/game/zz_windowreacts.rpy

```diff
@@ -217,10 +230,11 @@ init python:
         #Otherwise, let's get the active window
         active_window = mas_getActiveWindow()
 
-        if non_inclusive:
-            return len([s for s in keywords if s.lower() in active_window]) > 0
-        else:
-            return len([s for s in keywords if s.lower() not in active_window]) == 0
+        return (
+            (len([s for s in keywords if s.lower() in active_window]) > 0)
+            if non_inclusive else
+            (len([s for s in keywords if s.lower() not in active_window]) == 0)
+        )
```

Comment:
this looks more confusing than what it was previously. the lines here are just too long for `x if condition else y`